aviary/subsystems/mass/test/test_flops_mass_builder.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+
+import numpy as np
+import openmdao.api as om
+
+from openmdao.utils.assert_utils import assert_check_partials, assert_near_equal
+
+from aviary.subsystems.mass.mass_builder import CoreMassBuilder
+from aviary.variable_info.enums import LegacyCode
+from aviary.variable_info.variables import Aircraft
+from aviary.variable_info.variable_meta_data import _MetaData as BaseMetaData
+import aviary.api as av
+
+FLOPS = LegacyCode.FLOPS
+
+
+class TestFLOPSMassBuilder(av.TestSubsystemBuilderBase):
+    """
+    That class inherits from TestSubsystemBuilder. So all the test functions are
+    within that inherited class. The setUp() method prepares the class and is run
+    before the test methods; then the test methods are run.
+    """
+
+    def setUp(self):
+        self.subsystem_builder = CoreMassBuilder(
+            'test_core_mass', meta_data=BaseMetaData, code_origin=FLOPS)
```

Comment:
These tests don't do anything, aside from verifying that the builder runs without a syntax error.

Also, all the set_aviary_values stuff doesn't do anything at all.

Reply:
Do you think it is not needed?

Reply:
TestSubsystemBuilderBase has 20 tests that should run

Reply:
Oh, this is a derived class with tests. Never mind then, it will run those.